tgui/packages/tgui/interfaces/XenomorphExtractor.jsx

```diff
@@ -0,0 +1,140 @@
+import { useBackend, useLocalState } from '../backend';
+import { Section, Button, Stack, NoticeBox, LabeledList, Flex, Box, Dropdown, Divider } from '../components';
+import { Window } from '../layouts';
+
+export const XenomorphExtractor = (_props, context) => {
+  const { act, data } = useBackend(context);
+
+  const {
+    organ,
+    points,
+    upgrades,
+    caste,
+    value,
+    categories,
+    current_clearance,
+    is_x_level,
+  } = data;
+  const dropdownOptions = categories;
+  const [selectedTab, setSelectedTab] = useLocalState('category', 'NONE');
```

Comment:
```suggestion
  const [selectedTab, setSelectedTab] = useState('NONE');
```
Also useLocalState will be depreciated. `useState` instead.
https://react.dev/reference/react/useState